src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.ts

```diff
@@ -1,22 +1,19 @@
 import { Component, OnDestroy, OnInit } from '@angular/core';
-
-import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
 import { detect } from 'detect-browser';
 import { Subscription } from 'rxjs';
-
 import { UserService } from '~/app/shared/api/user.service';
 import { AppConstants, USER, VERSION_PREFIX } from '~/app/shared/constants/app.constants';
 import { LocalStorage } from '~/app/shared/enum/local-storage-enum';
 import { Permission } from '~/app/shared/models/permissions';
 import { AuthStorageService } from '~/app/shared/services/auth-storage.service';
 import { SummaryService } from '~/app/shared/services/summary.service';
-
+import { CdForm } from '~/app/shared/forms/cd-form';
 @Component({
   selector: 'cd-about',
   templateUrl: './about.component.html',
   styleUrls: ['./about.component.scss']
 })
-export class AboutComponent implements OnInit, OnDestroy {
+export class AboutComponent extends CdForm implements OnInit, OnDestroy {
```

Comment:
This is not a form so you shouldn't extend it to the CdForm and instead directly to the `BaseModal` component of carbon.
```suggestion
export class AboutComponent extends BaseModal implements OnInit, OnDestroy {
```

Reply:
The suggested changes have been made